lib/trino-filesystem-alluxio/src/test/java/io/trino/filesystem/alluxio/TestAlluxioFileSystem.java

```diff
@@ -0,0 +1,95 @@
+
+    @Container
+    private static final GenericContainer<?> ALLUXIO_MASTER_CONTAINER = createAlluxioMasterContainer();
+
+    @Container
+    private static final GenericContainer<?> ALLUXIO_WORKER_CONTAINER = createAlluxioWorkerContainer();
+
+    private static GenericContainer<?> createAlluxioMasterContainer()
+    {
+        GenericContainer<?> container = new GenericContainer<>(ALLUXIO_IMAGE);
+        container.withCommand("master-only")
+                .withEnv("ALLUXIO_JAVA_OPTS",
+                        "-Dalluxio.security.authentication.type=NOSASL "
+                                + "-Dalluxio.master.hostname=localhost "
+                                + "-Dalluxio.worker.hostname=localhost "
+                                + "-Dalluxio.master.mount.table.root.ufs=/opt/alluxio/underFSStorage "
+                                + "-Dalluxio.master.journal.type=NOOP "
+                                + "-Dalluxio.security.authorization.permission.enabled=false "
+                                + "-Dalluxio.security.authorization.plugins.enabled=false ")
+                .withNetworkMode("host")
+                .withAccessToHost(true)
+                .waitingFor(Wait.forLogMessage(".*Primary started*\n", 1));
+        return container;
+    }
+
+    private static GenericContainer<?> createAlluxioWorkerContainer()
+    {
+        GenericContainer<?> container = new GenericContainer<>(ALLUXIO_IMAGE);
+        container.withCommand("worker-only")
+                .withNetworkMode("host")
+                .withEnv("ALLUXIO_JAVA_OPTS",
+                        "-Dalluxio.security.authentication.type=NOSASL "
+                                + "-Dalluxio.worker.ramdisk.size=128MB "
+                                + "-Dalluxio.worker.hostname=localhost "
+                                + "-Dalluxio.worker.tieredstore.level0.alias=HDD "
+                                + "-Dalluxio.worker.tieredstore.level0.dirs.path=/tmp "
+                                + "-Dalluxio.master.hostname=localhost "
+                                + "-Dalluxio.security.authorization.permission.enabled=false "
+                                + "-Dalluxio.security.authorization.plugins.enabled=false ")
+                .withAccessToHost(true)
+                .dependsOn(ALLUXIO_MASTER_CONTAINER)
+                .waitingFor(Wait.forLogMessage(".*Alluxio worker started.*\n", 1));
+        return container;
+    }
+
+    @BeforeAll
+    void setup()
+            throws IOException
+    {
+        initialize();
+        // the SSHD container will be stopped by TestContainers on shutdown
+        // https://github.com/trinodb/trino/discussions/21969
+        System.setProperty("ReportLeakedContainers.disabled", "true");
+    }
+
+    @Test
+    void testContainer()
+    {
+        assertThat(ALLUXIO_MASTER_CONTAINER.isRunning()).isTrue();
+        assertThat(ALLUXIO_WORKER_CONTAINER.isRunning()).isTrue();
```

Comment:
What's the motivation to verify containers' status here?

Reply:
We can remove it in a later PR. it's mostly for verify the docker status during the test